CMakeLists.txt

```diff
@@ -12,3 +12,9 @@ set(addon_inc ${CMAKE_CURRENT_SOURCE_DIR}/tsp/inc)
 
 COMPILELIB("")
 
+
+file(GLOB_RECURSE MAC "${CMAKE_CURRENT_SOURCE_DIR}/macros/*")
+INSTALL(FILES ${MAC} DESTINATION ./macros/track)
+
+ADD_LIBRARY_TEST()
```

Comment:
@cmargalejo The pipeline fails because of this line, this library currently doesn't have any tests added so this causes failure when it doesn't find them.

```suggestion
```